src/hammer-vlsi/defaults.yml

```diff
@@ -15,6 +15,11 @@ vlsi.core:
   # TODO: move this to vlsi.technology (or technology.core) ucb-bar/hammer#317
   node: 32
 
+  # Build system to use, if any.
+  # This is used to auto-generate a build file, if desired. It is not required to run hammer.
+  # Valid options are "none" or "make"
+  build_system: "make"
```

Comment:
Since it's not required maybe we could leave the default as `none`?

Reply:
I can do that.